styles/yarn.lock

```diff
@@ -3,1605 +3,1802 @@
 
 
 "@babel/code-frame@^7.0.0":
```

Comment:
What was the reason for alll of these version bumps?  Might make sense for this to be in a separate PR.